deps.windows/60-python.ps1

```diff
@@ -2,10 +2,11 @@ param(
     [string] $Name = 'python',
     [string] $Version = '3.1.1',
     [string] $Uri = 'https://github.com/pyenv-win/pyenv-win.git',
-    [string] $Hash = '754a6ca877f66aaa2bd4485a29411267d0705273',
+    [string] $Hash = '3201b6c2bf1e4d4791f66fc8634e80f3ff2731c6',
```

Comment:
Is this just to get a newer pyenv-win cache?

Reply:
That's required to get a Pyenv version that supports Python 3.11 and Windows on ARM at all.

Reply:
You can also (or should be able to) refresh the cache if it's just about updating the cache of available Python versions. If there's something specific to the pyenv-win code itself though, that's worth noting.

Reply:
Afaik the Pyenv version itself didn't have support for it.

Reply:
This is probably fine. While I'd prefer that pyenv-win had a new tag, not much we can do there.